recipes/libsodium/libsodium.inc

```diff
@@ -0,0 +1,8 @@
+inherit autotools c pkgconfig
```

Comment:
You should probably inherit "library" class also.  It is designed for single-library recipes such as this.